<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgAnneeNuevo</class>
 <widget class="QWidget" name="dlgAnneeNuevo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Día</string>
       </property>
       <property name="buddy">
        <cstring>spDia</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spDia">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>31</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Mes</string>
       </property>
       <property name="buddy">
        <cstring>spMes</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spMes">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>12</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btAnadirPersonaPrincipal">
       <property name="text">
        <string>Persona principal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPersonaPrincipal">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Tema principal</string>
       </property>
       <property name="buddy">
        <cstring>txtTemaPrincipal</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtTemaPrincipal"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Pensamiento</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTextEdit" name="txtPensamiento"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Autor</string>
             </property>
             <property name="buddy">
              <cstring>txtAutorPensamiento</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtAutorPensamiento"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Entradilla</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QTextEdit" name="txtEntradilla"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Final thought</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QTextEdit" name="txtFinalThought"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Personas extra</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="btAnadirPersonaAdicional">
           <property name="text">
            <string>Persona adicional</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btQuitarPersonaAdicional">
           <property name="text">
            <string>Quitar</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="twPersonasAdicionales"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Meditaciones</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QSpinBox" name="spNumeroMeditacion"/>
           </item>
           <item>
            <widget class="QLineEdit" name="txtMeditacion"/>
           </item>
           <item>
            <widget class="QPushButton" name="btTemas">
             <property name="text">
              <string>Temas</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btAnadirMeditacion">
             <property name="text">
              <string>Añadir</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btQuitarMeditacion">
             <property name="text">
              <string>Quitar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="twMeditaciones"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btCancelar">
       <property name="text">
        <string>Can&amp;celar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btOK">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
